backend/python/tests/functional/test_transport_method_service.py

```diff
@@ -48,11 +48,35 @@ def test_get_transport_method_raises_exception_null_argument(
     transport_method_service,
 ):
     with pytest.raises(Exception):
-        transport_method_service._get_transport_method(None)
+        transport_method_service.get_transportation_method(None)
 
 
 def test_get_transport_method_raises_exception_invalid_argument(
     transport_method_service,
 ):
     with pytest.raises(Exception):
-        transport_method_service._get_transport_method(12345)
+        transport_method_service.get_transportation_method(12345)
+
+
+def test_get_transportation_methods_success(transport_method_service):
+    res = transport_method_service.get_transportation_methods()
+    # Check if list is returned
+    assert type(res) == list
+    # Check if returned list is not empty
+    assert res
+    # Check if returned list has TransportationMethodDTOs
+    assert type(res[0]) is TransportationMethodDTO
+    # Check if correct number of items are returned
+    assert len(res) == 3
+    # Check values of all items
+    transportation_methods = set(
+        [key["transportation_method"] for key in DEFAULT_TRANSPORTATION_METHODS]
+    )
+    for transportation_method_object in res:
+        assert (
+            transportation_method_object.transportation_method in transportation_methods
+        )
+        transportation_methods.remove(
+            transportation_method_object.transportation_method
+        )
+    assert len(transportation_methods) == 0
```

Comment:
thoughts on something like this instead? it should do the same checks but i think this is a bit more understandable (feel free to modify though!)

also i don't think we need to comment exactly what we're doing unless it's really unclear (e.g. you could add a comment like `Checking return matches expected transportation methods`, but other ones i don't think are necessary)

```suggestion
    assert type(res) == list
    assert len(res) == 3
    assert all(type(item) == TransportationMethodDTO for item in res)

    transportation_methods_db = [
        entry["transportation_method"] for entry in DEFAULT_TRANSPORTATION_METHODS
    ]
    transportation_methods_res = [item.transportation_method for item in res]
    assert set(transportation_methods_db) == set(transportation_methods_res)

```

Reply:
Yes, definitely increases readability! Will commit the changes.